__tests__/dotnet.test.ts

```diff
@@ -6,6 +6,20 @@ import { Platform } from '../src/platform';
 import { ToolsDirectory } from '../src/toolsDirectory';
 
 const targetDirectory = path.join('target', 'directory');
+const cakeVersion = '0.35.0';
+const installedCakeVersion = '0.34.0';
+const containsFileWithOldToolInstalled = (path: string) => {
```

Comment:
These variables belong to a specific context (`When installing the Cake Tool locally to a directory where it already exists`), so it would make sense to define them there instead of the global scope.

Reply:
`containsFileWithOldToolInstalled` is used in `'When failing to install the Cake Tool locally` too.
`containsFileWithOldToolInstalled` uses `installedCakeVersion`.